packages/cli/src/CredentialsHelper.ts

```diff
@@ -695,7 +695,7 @@ export class CredentialsHelper extends ICredentialsHelper {
 							`Received HTTP status code: ${errorResponseData.statusCode}`,
 					};
 				}
-			} else if (error.cause.code) {
+			} else if (error.cause?.code) {
```

Comment:
[error.cause](https://nodejs.org/api/errors.html#errorcause) was added in Node 16 and it throws 500 in Node 14.